lib/ClangImporter/ImporterImpl.h

```diff
@@ -108,8 +108,8 @@ enum class ImportTypeKind {
   /// \brief Import the type of a literal value.
   Value,
 
-  /// \brief Import the type of a literal value that can be bridged.
-  BridgedValue,
+  /// \brief Import the type of an Objective-C generic argument.
+  ObjCGenericArgument,
```

Comment:
I feel like this isn't just *any* generic argument, since most of the time we'd import `Foo<NSString *> *` as `Foo<NSString>` and not `Foo<String>`. "ObjCCollectionElement", maybe?

Reply:
Good point!